slo-workload/src/Commands/RunCommand.php

```diff
@@ -0,0 +1,379 @@
+            "type" => "int",
+            "description" => "prometheus push period in milliseconds"
+        ],
+        [
+            "alias" => ["-read-rps"],
+            "type" => "int",
+            "description" => "read RPS"
+        ],
+        [
+            "alias" => ["-read-timeout"],
+            "type" => "int",
+            "description" => "read timeout milliseconds"
+        ],
+        [
+            "alias" => ["-write-rps"],
+            "type" => "int",
+            "description" => "write RPS"
+        ],
+        [
+            "alias" => ["-write-timeout"],
+            "type" => "int",
+            "description" => "write timeout milliseconds"
+        ],
+        [
+            "alias" => ["-time"],
+            "type" => "int",
+            "description" => "run time in seconds"
+        ],
+        [
+            "alias" => ["-shutdown-time"],
+            "type" => "int",
+            "description" => "graceful shutdown time in seconds"
+        ]
+    ];
+    public $help = "run <endpoint> <db> [options]
+Arguments:
+  endpoint                        YDB endpoint to connect to
+  db                              YDB database to connect to
+
+Options:
+  -t -table-name         <string> table name to create
+
+  -initial-data-count    <int>    amount of initially created rows
+
+  -prom-pgw              <string> prometheus push gateway
+  -report-period         <int>    prometheus push period in milliseconds
+
+  -read-rps              <int>    read RPS
+  -read-timeout          <int>    read timeout milliseconds
+
+  -write-rps             <int>    write RPS
+  -write-timeout         <int>    write timeout milliseconds
+
+  -time                  <int>    run time in seconds
+  -shutdown-time         <int>    graceful shutdown time in seconds";
+    /**
+     * @var int
+     */
+    protected $queueId;
+
+    public function execute(string $endpoint, string $path, array $options)
+    {
+        $startTime = microtime(true);
+        $tableName = $options["-table-name"] ?? Defaults::TABLE_NAME;
+        $initialDataCount = (int)($options["-initial-data-count"] ?? Defaults::GENERATOR_DATA_COUNT);
+        $promPgw = ($options["-prom-pgw"] ?? Defaults::PROMETHEUS_PUSH_GATEWAY);
+        $reportPeriod = (int)($options["-report-period"] ?? Defaults::PROMETHEUS_PUSH_PERIOD);
+        $readForks = ((int)($options["-read-rps"] ?? Defaults::READ_RPS)) / Defaults::RPS_PER_READ_FORK;
+        $readTimeout = (int)($options["-read-timeout"] ?? Defaults::READ_TIMEOUT);
+        $writeForks = ((int)($options["-write-rps"] ?? Defaults::WRITE_RPS)) / Defaults::RPS_PER_WRITE_FORK;
+        $writeTimeout = (int)($options["-write-timeout"] ?? Defaults::WRITE_TIMEOUT);
+        $time = (int)($options["-time"] ?? Defaults::READ_TIME) - 5;
+        $shutdownTime = (int)($options["-shutdown-time"] ?? Defaults::SHUTDOWN_TIME);
+
+        $this->queueId = ftok(__FILE__, 'm');
+        $msgQueue = msg_get_queue($this->queueId);
```

Comment:
need call msg_remove_queue before msg_remove_queue - for prevent reuse queue from previous start (if script will called many times).